utils/linkage.py

```diff
@@ -135,6 +270,77 @@ def get_street_from_address_line_1(address_line_1: str) -> str:
     return " ".join(string)
 
 
+def convert_duplicates_to_dict(df: pd.DataFrame) -> pd.DataFrame:
+    """Saves to the "output" directory a file mapping multiple strings to one
+    string
+
+    Given a dataframe where each row contains one string in a column and a list
+    of strings in another column, the function maps each string in the list to
+    the single string.
+
+    Args:
+        A pandas dataframe
+
+    Returns
+        None. However it outputs a file to the output directory, with 2
+        columns. The first, which indicates the duplicated UUIDs, is labeled
+        'duplicated_uuids', and the 2nd, which shows the uuids to which the
+        deduplicated entries match to, is labeled 'mapped_uuids'.
+    """
+    deduped_dict = {}
+    for i in range(len(df)):
+        deduped_uudis = df.iloc[i]["duplicated"]
+        for j in range(len(deduped_uudis)):
+            deduped_dict.update({deduped_uudis[j]: df.iloc[i]["id"]})
+
+    # now convert dictionary into a csv file
+    deduped_df = pd.DataFrame.from_dict(deduped_dict, "index")
+    deduped_df = deduped_df.reset_index().rename(
+        columns={"index": "duplicated_uuids", 0: "mapped_uuids"}
+    )
+    deduped_df.to_csv(
```

Comment:
if you add in the arg "header=false" I think this resolved the duplicated column name rows